packages/devtools_app/lib/src/memory/flutter/memory_heap_tree_view.dart

```diff
@@ -537,6 +620,97 @@ class HeapTreeViewState extends State<HeapTree> with AutoDisposeMixin {
     );
   }
 
+  bool _classMatcher(HeapGraphClassLive liveClass) {
+    final regExp = RegExp(knownClassesRegExs);
+    return regExp.allMatches(liveClass.name).isNotEmpty;
+  }
+
+  List<ExternalReference> _collect(List dupSearch) {
```

Comment:
specify the type of List that dupSearch is.

Reply:
This code was eliminated.